### ISSUE TITLE
CoqIDE: wrong highlighting of error msgs?

### ISSUE THREAD

**Issue by orilahav** [coq/coq#13064]
#### Description of the problem

"No applicable tactic" highlights an applicable tactic (a later non-applicable tactic exists).

![Screen Shot 2020-09-22 at 9 45 04](https://user-images.githubusercontent.com/14070781/93851387-b2131b00-fcb8-11ea-8375-3a689a34ab39.png)
![Screen Shot 2020-09-22 at 9 45 24](https://user-images.githubusercontent.com/14070781/93851392-b3444800-fcb8-11ea-9719-b00ae0ea6257.png)

#### Coq Version

8.12.0

**Comment by corwin-of-amber** [comment]
I tested with `coqtop` (8.12.0) and I also see `simpl` as the error location.
```
Coq < Lemma aaa : 1 + 1 = 2.
1 subgoal

  ============================
  1 + 1 = 2

aaa < simpl; discriminate.
Toplevel input, characters 0-5:
> simpl; discriminate.
> ^^^^^
Error: Tactic failure: No applicable tactic.

aaa <
```

**Comment by ejgallego** [comment]
Fixed in 8.12.1 , duplicate of #12773
